src/rpc/blockchain.cpp

```diff
@@ -406,6 +409,23 @@ void entryToJSON(UniValue &info, const CTxMemPoolEntry &e)
     }
 
     info.pushKV("depends", depends);
+
+    std::set<uint256> set_spent;
+    UniValue spent(UniValue::VARR);
+    for (unsigned int i = 0; i < tx.vout.size(); i++) {
```

Comment:
This for loop iterates tx.vout, does it equal to:
`auto entryit = mempool.mapTx.find(tx.GetHash());`
`auto children = mempool.GetMemPoolChildren(entryit);`
?

Reply:
The index is needed below, to construct the outpoint from the output.

Reply:
Right, I've overlooked @junfx suggestion.